File: Lat2/C++/Hardware.cpp
#include <iostream>
#include <string>
using namespace std;
#include "Product.cpp"
class Hardware : public Product{
    private:
        string brand;
        string model;
    public:
    Hardware(){}
    void setBrand(string brand){
        this->brand = brand;
    }
    string getBrand(){
        return this->brand;
    }
    void setModel(string model){
        this->model = model;
    }
    string getModel(){
        return this->model;
    }
    ~Hardware(){}

};

File: Lat2/Python/Hardware.py
from statistics import mode
from Product import Product
class Hardware(Product):
    def __init__(self):
        self.brand = ""
        self.model = ""
    def setBrand(self, brand):
        self.brand = brand
    def getBrand(self):
        return self.brand
    def setModel(self, model):
        self.model = model
    def getModel(self):
        return self.model